stock_reorder_forecast/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock reorder forecast",
+    "version": "9.0.1.0.0",
+    "author": "Therp BV",
```

Comment:
Add OCA to authors in the usual way: "Therp BV,Odoo Community Association (OCA)"